### PR TITLE
code cleanup and adding randBytes()

### PR DESCRIPTION
**Pull request by BaoshanPang**
This is to address:
1. https://github.com/rust-lang/libc/issues/1469
2. https://github.com/rust-random/getrandom/pull/86#issuecomment-521440946

### CODE REVIEW

File: src/vxworks/mod.rs [hunk 1]
```diff
@@ -2167,6 +2016,9 @@ extern {
         iov: *const ::iovec,
         iovcnt: ::c_int,
     ) -> ::ssize_t;
+
+    // randomNumGen.h
+    pub fn randBytes(buf: *mut u8, length: i32) -> i32;
```
Comment (newpavlov) [inline src/vxworks/mod.rs]: For consistency you should use `pub fn randBytes(buf: *mut ::c_uchar, length: ::c_int) -> c_int` (or whatever types you have in `randomNumGen.h`).

**Comment by n-salim** [comment]
Let's add randABytes and randUBytes too. I need to check which of these is the best to use in the rand crate.

**Comment by BaoshanPang** [comment]
> Let's add randABytes and randUBytes too. I need to check which of these is the best to use in the rand crate.

Sure.

**Review by n-salim — COMMENTED** [review]
Also let's cleanup the comment right at the top of the file:

`//! Hacking together the definitions for VxWorks Bindings`


Suggest replacing with:

`//! Interface to VxWorks C library`

File: src/vxworks/mod.rs [hunk 2]
```diff
@@ -2167,6 +2018,12 @@ extern {
         iov: *const ::iovec,
         iovcnt: ::c_int,
     ) -> ::ssize_t;
+
+    // randomNumGen.h
+    pub fn randBytes(buf: *mut c_uchar, length: c_int) -> c_int;
+    pub fn randABytes(buf: *mut c_uchar, length: c_int) -> c_int;
+    pub fn randUBytes(buf: *mut c_uchar, length: c_int) -> c_int;
+    pub fn randSecure() -> BOOL;
```
Comment (BaoshanPang) [inline src/vxworks/mod.rs]: @n-salim 
Should I change BOOL to c_int ? althoug the BOOL is defined as c_int here.

Comment (n-salim) [reply src/vxworks/mod.rs]: Yes make it c_int to align to the proposed future signature.

Comment (BaoshanPang) [reply src/vxworks/mod.rs]: Sure.

**Review by n-salim — APPROVED** [review]

**Pull request merged**